mne_bids/write.py

```diff
@@ -965,21 +967,33 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     _mkdir_p(os.path.dirname(bids_fname))
 
     if verbose:
-        print('Copying data files to %s' % bids_fname)
-
-    if ext not in ALLOWED_EXTENSIONS:
-        raise ValueError('ext must be in %s, got %s'
-                         % (''.join(ALLOWED_EXTENSIONS), ext))
+        print('Copying data files to %s' % op.splitext(bids_fname)[0])
 
+    convert = ext not in ALLOWED_EXTENSIONS[kind]
     # Copy the imaging data files
-    if ext in ['.fif']:
+    if convert:
+        if kind == 'meg':
+            raise ValueError('Got file extension %s for MEG data, ' +
+                             'expected one of %s' % ALLOWED_EXTENSIONS['meg'])
+        if verbose:
+            warn('Converting data files to BrainVision format')
+        if not check_version('pybv', '0.2'):
+            raise ImportError('pybv is required for this conversion')
```

Comment:
Sorry last (I promise!) comment. Can you make this a more informative error?

```suggestion
            raise ImportError('pybv >=0.2.0 is required for converting %s files to Brainvision format' % ext)
```